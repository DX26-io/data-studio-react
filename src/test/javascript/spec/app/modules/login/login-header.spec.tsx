import React from 'react';
import { shallow } from 'enzyme';
import LoginHeader from 'app/modules/login/login-header';

describe('Login Top Text', () => {
  let mountedWrapper;

  const wrapper = () => {
    if (!mountedWrapper) {
      mountedWrapper = shallow(<LoginHeader />);
    }
    return mountedWrapper;
  };

  beforeEach(() => {
    mountedWrapper = undefined;
  });

  it('Renders Login Top Text', () => {
    const component = wrapper();
    expect(component).toMatchSnapshot();
  });
});
