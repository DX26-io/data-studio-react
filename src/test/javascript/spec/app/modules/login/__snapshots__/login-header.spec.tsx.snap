// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Top Text Renders Login Top Text 1`] = `
<ForwardRef($e9098d18abf2a118bc3bc95f0d402bb5$var$Flex)
  direction="row-reverse"
  margin="size-600"
>
  <ForwardRef($fe1955053990ab873941970c7463$var$Text)>
    <span
      className="spectrum-Heading spectrum-Heading--sizeL"
    >
      Dx26
    </span>
  </ForwardRef($fe1955053990ab873941970c7463$var$Text)>
</ForwardRef($e9098d18abf2a118bc3bc95f0d402bb5$var$Flex)>
`;
